README.md

```diff
@@ -44,13 +45,18 @@ If you're a designer and would like to contribute, feel free to reach out to us!
 The app is built with the following code dependencies:
 
 1. [NextJS](https://github.com/vercel/next.js)
+2. [Express](https://github.com/expressjs/express)
+3. [PostgreSQL](https://github.com/postgres/postgres)
```

Comment:
Since this is open, could we add Styled Components? Seeing how we are using it on the Front-End.